packages/calcite-components/src/components/dialog/dialog.e2e.ts

```diff
@@ -1160,4 +1160,109 @@ describe("calcite-dialog", () => {
       expect(await dialog.getProperty("open")).toBe(true);
     });
   });
+
+  describe("focusTrap", () => {
+    let page: E2EPage;
+
+    beforeEach(async () => {
+      page = await newE2EPage();
+      await page.setContent(html`
+        <calcite-dialog width-scale="s" open closable><button id="insideEl">inside</button></calcite-dialog>
+        <button id="outsideEl">outside</button>
+      `);
+
+      await skipAnimations(page);
+      await page.waitForChanges();
+    });
+
+    it("can tab out of non-modal dialog when focusTrapDisabled=true", async () => {
```

Comment:
It'd be helpful to group these in describe blocks (a group for modal and non-modal OR a group for focusTrapDisabled when true and false).

Reply:
Yeah, I thought of it the same way and because it could group either way, opted for leaving them all in one describe block. I'll group them for modal and non-modal though, since that drives whether focusTrapDisabled takes effect or not.